components/buttons/queue.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
 		}
 		if (!player || page < 1 || page > pages.length) {
 			const original = interaction.message.components;
-			original[0].components.forEach(c => c.setDisabled(true));
+			original[0].components.map(c => ButtonBuilder.from(c).setDisabled(true));
```

Comment:
This condition did nothing.

https://user-images.githubusercontent.com/18362871/179450345-4e0b410c-d82c-4b5b-bcd6-a7471e49654d.mp4